db/migrate/20230313124749_add_carehome_charges_value_check.rb

```diff
@@ -0,0 +1,5 @@
+class AddCarehomeChargesValueCheck < ActiveRecord::Migration[7.0]
+  def change
+    add_column :lettings_logs, :carehome_charges_value_check, :integer
```

Comment:
Do we need to backdate values for this?

Reply:
I don't think so. I don't think it is currently possible to have the answers to `is_carehome` as "Yes" and "chcharge" as nil, because it would throw a validation (?)